### DISCUSSION TITLE
Rename open / private groups

### DISCUSSION THREAD

**Discussion by peepo5** [oxen-io/session-desktop#2090] [Ideas]
I think it may be confusing for new users to understand what private/open groups mean. Private groups are not strictly private, and the main difference is that they run on the network, compared to open groups which are hosted. I think that new names for these types may be better.

Ideas: 

Maybe rename closed groups to just "groups" or make a new name
Also open groups could be renamed to a new name

Let me know what you guys think ;P

**Comment by Bilb** [comment]
open groups were named channels at some point and I believe they are better named like this.
I like that they have their names tell how safe you are on each.

**Comment by KeeJef** [comment]
Open groups are now "Communities" and Closed groups are just "Groups"
